mathics/builtin/arithmetic.py

```diff
@@ -418,7 +418,6 @@ class Complex_(Builtin):
     <dl>
       <dt>'Complex'
       <dd>is the head of complex numbers.
-
```

Comment:
Space between groups of <dt> was deliberate.  Is it too hard to preserve this?

Reply:
OK. It is almost trivial to restore them. Please, when you have time, check the rules I have used, to see if they need to be adjusted.

Reply:
Sure - will do! Probably this weekend after some of the other backlogged items.